public/scripts/templates/importCharacters.html

```diff
@@ -19,7 +19,7 @@ <h4 data-i18n="Enter the URL of the content to import">Enter the URL of the cont
     <span data-i18n="Write each URL or ID into a new line.">
         Write each URL or ID into a new line.
     </span>
-    <span data-i18n="(Shift + Enter to start a new line)">
-        (Shift + Enter to start a new line)
+    <span data-i18n="(&lt;kbd&gt;Shift&lt;/kbd&gt; + &lt;kbd&gt;Enter&lt;/kbd&gt; to start a new line)">
```

Comment:
Text inserted from i18n is escaped so this will not work

Reply:
What do I have to do to "make this work"? Switch it into separate `<span>` blocks for the text too? :/

Reply:
Remove this passage altogether? I don't think that power users would have problems with inserting a newline. Given that not even the main text input has this prompt.

Reply:
Sure boss. 🫡